pepper-apis/src/main/java/org/broadinstitute/ddp/db/dto/ActivityInstanceDto.java

```diff
@@ -95,10 +95,6 @@ public Long getFirstCompletedAt() {
         return firstCompletedAt;
     }
 
-    public boolean isReadonly() {
-        return Optional.ofNullable(isReadonly).orElse(false);
```

Comment:
I removed this method so we don't default read-only to `false` and cause another issue with bypassing check.